src/backend/src/services/organizations.service.ts

```diff
@@ -53,4 +53,27 @@ export default class OrganizationsService {
 
     return newLinks;
   }
+
+  /**
+    Gets all the useful links for an organization
+    @param organizationId the organization to get the links for
+    @returns the useful links for the organization
+  */
+  static async getAllUsefulLinks(organizationId: string) {
+    const organization = await prisma.organization.findUnique({
+      where: { organizationId },
+      select: { usefulLinks: { select: { linkId: true } } }
+    });
+
+    if (!organization) {
```

Comment:
Missed this the first time through -- change this to be a `NotFoundException` and this is good to go